Client/Blogger/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,130 @@
+import "../Styles/Navigation_styles/Navigations.css";
+import { NavLink, useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { useCookies } from "react-cookie";
+import { resetUserState } from "../Redux/Feature/userSlice";
+import { useRef, useState } from "react";
+
+function NavigationBar() {
+  const [showNav, setShowNav] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const navPostion = {
+    display: "flex",
+    justifyContent: "space-around",
```

Comment:
Before proceeding with a code review, always check for unused/unwanted code and remove it. Here, it seems that this code (navPostion) not being used anywhere